vg/vgimg/vgimg_test.go

```diff
@@ -0,0 +1,48 @@
+package vgimg_test
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/gonum/plot"
+	"github.com/gonum/plot/plotter"
+	"github.com/gonum/plot/vg"
+	"github.com/gonum/plot/vg/draw"
+	"github.com/gonum/plot/vg/vgimg"
+)
+
+func TestIssue179(t *testing.T) {
+	scatter, err := plotter.NewScatter(plotter.XYs{{1, 1}, {0, 1}, {0, 0}})
+	if err != nil {
+		log.Fatal(err)
+	}
+	p, err := plot.New()
+	if err != nil {
+		log.Fatal(err)
+	}
+	p.Add(scatter)
+	p.HideAxes()
+
+	c := vgimg.JpegCanvas{Canvas: vgimg.New(5.08*vg.Centimeter, 5.08*vg.Centimeter)}
```

Comment:
I'm not sure which is more sad. The number of times I was given numbers like this in undergrad engineering classes, or the number of times I had to use the alternating units of pound-mass and slugs.